### ISSUE TITLE
Visualisation of 3D Array in Julia

### ISSUE THREAD

**Issue by fz920** [BrianChung10/SOP#13]
Hi @dlfivefifty @ioannisPApapadopoulos, about the finite difference in 3D we talked about today, now I have computed the solution which is a 3d array. I wonder if there is any Julia function that you would recommend I use to visualize the solution? Thanks!

**Comment by dlfivefifty** [comment]
The easy answer: take 2D _slices_ of the 3D function and plot those.

**Comment by fz920** [comment]
> The easy answer: take 2D _slices_ of the 3D function and plot those.

Thank you for your answer. Suppose now I have a solution which is a 9x9x9 array, say A. Do you mean I should take A[:, :, 1], A[:, :, 2], ..., A[:, :, 9] and plot 9 contours for these 2D slices?

**Comment by dlfivefifty** [comment]
Yes, though you wouldn't necessarily need to plot _every_ slice.

There are more sophisticated plots e.g.  something like:

<img width="330" alt="image" src="https://user-images.githubusercontent.com/1229737/174268541-e4793e48-c71b-4046-8469-37d5b4a17cdf.png">

I don't think it's worth it though. You would probably need to use WriteVTK.jl to generate the data and then use ParaView. Which are things I've never done myself.
